draft-ietf-rats-corim.md

```diff
@@ -1171,7 +1177,15 @@ If the search criteria are satisfied, the `endorsements` entries are asserted wi
 
 #### Conditional Endorsement Series Triple {#sec-comid-triple-cond-series}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/310
+A Conditional Endorsement Series triple uses a stateful environment, (i.e. `stateful-environment-record`), that identifies a Target Environment based on an `environment-map` plus the `measurement-map` measurements that have matching Evidence.	
```

Comment:
This reference to the internals is not in tune with the "gentle introduction" spirit of this PR.

```suggestion
A Conditional Endorsement Series triple uses a "stateful environment" that identifies a Target Environment plus the measurements that have matching Evidence.	
```

Reply:
I think the orginal text is more clear as the triple is indeed very complex so more the text clarity the merrier it is!

Reply:
Also `` is the convention we have been following, so prefer this way as against to a new convention of "xyz"

Reply:
If using the actual CDDL (which occurs earlier in the document?) can't be used here, the reader is equally confused by "stateful environment" as they would be `stateful-environment`. The current text assumes the reader is familiar with what has been discussed in previous sections. 

The goal of #310 is to provide one or two sentences of elevator text that describes the triple subject, object, and predicate without using those terms.

Reply:
Ned to address this in PR #343